README.md

```diff
@@ -185,3 +185,59 @@ sudo: true
 before_script:
   - sudo sysctl -w vm.max_map_count=262144
 ```
+
+**Avoid downloading internet again and again**: When you want to run integration tests with a given plugin like `x-pack` 
+for example, `maven-elasticsearch-plugin` will run behind the scene a command like `bin/elasticsearch install x-pack` 
+which will download again and again the plugin from internet.
+
+You can use some Maven magic to avoid that download times by using first `maven-dependency-plugin` to download the plugin 
+as an artifact which will be stored first in your local `.m2` directory then downloaded from there to your project target directory.
```

Comment:
Can you please provide an example of how to download a plugin to ~/.m2 (such as the one in https://stackoverflow.com/a/1776808, to avoid googling)?

Please change `then download from there` to `then copy from there`.